OptimizelySDK/Bucketing/DecisionService.cs

```diff
@@ -351,15 +407,15 @@ public void SaveVariation(Experiment experiment, Variation variation, UserProfil
             try
             {
                 UserProfileService.Save(userProfile.ToMap());
-                Logger.Log(LogLevel.INFO, $"Saved variation \"{variation.Id}\" of experiment \"{experiment.Id}\" for user \"{userProfile.UserId}\".");
+                Logger.Log(LogLevel.INFO, reasons?.AddInfo($"Saved variation \"{variation.Id}\" of experiment \"{experiment.Id}\" for user \"{userProfile.UserId}\"."));
```

Comment:
This message should not be added into "reasons" (it's not a reason for the current decision).